### PR TITLE
As a user, I can download a CSV of user data

### PR DESCRIPTION
**Pull request by jordanmaguire**
Adding CSV download of current results in to be used in Top Club.

screen shot 2015-12-09 at 3 37 54 pm

Will get @pzi to style it when he is finished with Tongln.

Will teach frontier generators to generate CSV for models.

I think I'll add a "per page" toggle as well, so admins can set it up to 250 results. (EG: 20 (default), 50, 100, 250)

That way they can get a bunch of data in their CSVs.

Regarding feature tests - not too sure how to test this, as the rails server will automatically send this data as a CSV.

### CODE REVIEW

**Comment by psyren89** [comment]
Short and sweet :+1:

File: app/views/admin/members/index.html.haml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 %h1 Members Management
 
 %p= link_to "Add new member", new_admin_member_path, class: "btn"
+%p= link_to "Download CSV", UriQueryMerger.new(request.url, {format: :csv}).merge, class: "btn"
```
Comment (osiro) [inline app/views/admin/members/index.html.haml]: [MINOR] what do you think about leaving this guy in a view helper?

```ruby
def link_to_csv(html_options={ class: "btn"} })
  link_to("Download CSV", UriQueryMerger.new(request.url, {format: :csv}).merge, html_options)
end
```

Comment (jordanmaguire) [inline app/views/admin/members/index.html.haml]: sounds like a decent abstraction

**Comment by osiro** [comment]
a

a

**Pull request merged**